samples/display/cfb_custom_font/README.rst

```diff
@@ -0,0 +1,12 @@
+.. _cfb_custom_fonts:
+
+Custom Fonts
+############
+
+Overview
+********
+A simple example showing how to generate Character Framebuffer (CFB)
+font headers automatically at build time.
+
+This example generates a font with font elements for 6 sided dice from
+a PNG image.
```

Comment:
Add...
```
a PNG image, and then uses the generated header (``cfb_font_dice.h``)
to show the font elements on the display of a supported board.
```

Reply:
Build and run instructions would be nice.

Reply:
@dbkinder Thank you. I have added the suggested improvement to the general description of the sample application and added build instructions.